ihmc-state-estimation/src/main/java/us/ihmc/stateEstimation/humanoid/kinematicsBasedStateEstimation/PelvisLinearStateUpdater.java

```diff
@@ -369,6 +369,8 @@ public void updateRootJointPositionAndLinearVelocity()
       numberOfEndEffectorsFilteredByLoad.set(0);
       numberOfEndEffectorsFilteredByVelocity.set(0);
 
+      numberOfEndEffectorsTrusted.set(filterTrustedFeetBasedOnVelocity(numberOfEndEffectorsTrusted.getIntegerValue()));
```

Comment:
Move this before filtering by load, so if the foot is doing something dumb, it's removed from being trusted.